installer/PowerToysSetup/Product.wxs

```diff
@@ -1343,6 +1351,12 @@
                 Guid="$(var.CompGUIDPrefix)13"
                 Directory="Resource$(var.IdSafeLanguage)WindowsSettingsPluginFolder">
                 <File Id="Launcher_WindowsSettings_$(var.IdSafeLanguage)_File" Source="$(var.BinX64Dir)modules\launcher\Plugins\WindowsSettings\$(var.Language)\Microsoft.PowerToys.Run.Plugin.WindowsSettings.resources.dll" />
+            </Component>
+			<Component
+                Id="Launcher_TimeDate_$(var.IdSafeLanguage)_Component"
+                Guid="$(var.CompGUIDPrefix)14"
```

Comment:
Let's put it into the end and make it 18, or I think this will fail in the signed release CI.

Reply:
good catch. didn't see the number